lnwallet/commitment.go

```diff
@@ -693,7 +693,12 @@ func CoopCloseBalance(chanType channeldb.ChannelType, isInitiator bool,
 		theirBalance += initiatorDelta
 	}
 
-	return ourBalance, theirBalance
+	if ourBalance < 0 || theirBalance < 0 {
```

Comment:
Make clear in a comment that this is a sanity check and not supposed to be hit?

Reply:
Done

Reply:
_narrator: it was hit_